homeassistant/components/modern_forms/__init__.py

```diff
@@ -66,6 +119,7 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     if not hass.data[DOMAIN]:
```

Comment:
if not .... and then del what is not there ?

Reply:
If there is a None entry in the dict it will remove it.

Reply:
yes but is that something that can actual happen?